code/obj/machinery/manufacturer.dm

```diff
@@ -26,62 +27,93 @@ TYPEINFO(/obj/machinery/manufacturer)
+	/// How much power is consumed while active. This is determined automatically when the unit starts a production cycle
+	var/active_power_consumption = 0
 	var/panel_open = FALSE
+	/// The stage of dismantlement this machine is currently at.
 	var/dismantle_stage = 0
 	var/hacked = FALSE
 	var/malfunction = FALSE
-	var/electrified = 0 //! This is a timer and not a true/false; it's decremented every process() tick
+	/// This is a timer decremented every process() tick representing how long the machine will be electrified for.
+	/// If this is 0, then the machine is no longer electrified. Use src.is_electrified() to check if the machine is electrified.
+	var/time_left_electrified = 0
+	/// A turf or object which the manufacturer will attempt to output items into.
 	var/output_target = null
+	/// A list populated in New() which stores turfs which are nearby this manufacturer.
 	var/list/nearby_turfs = list()
-	var/wires = 15 //! This is a bitflag used to track wire states, for hacking and such. Replace it with something cleaner if an option exists when you're reading this :p
-	var/output_message_user = null //! Used as a cache when outputting messages to users
+	/// This is a bitflag used to track wire states, for hacking and such. Replace it with something cleaner if an option exists when you're reading this :p
+	var/wires = 15
 	var/frequency = FREQ_PDA
 	var/net_id = null
 	var/device_tag = "PNET_MANUFACTURER"
+	/// The data terminal attached underfloor to this manufacturer. Allows use of PNET packets
 	var/obj/machinery/power/data_terminal/link = null
-	var/obj/item/card/id/scan = null //! Used when deducting payment for ores from a Rockbox
-
-	// Printing and queues
-	var/original_duration = 0 //! duration of the currently queued print, used to keep track of progress when M.time gets modified weirdly in queueing
-	var/time_left = 0 //! time the current blueprint will take to manufacture
-	var/time_started = 0 //! time the last blueprint was queued
+	/// Card currently scanned into the machine, used when deducting payment for ores from a Rockbox
+	var/obj/item/card/id/scan = null
+
+	/* Printing and queues */
+	/// Original duration of the currently queued print, used to keep track of progress when M.time gets modified weirdly in queueing
+	var/original_duration = 0
+	/// Time left until the current blueprint is complete. Updated on pausing and on starting a new blueprint.
+	var/time_left = 0
+	/// Time the blueprint was queued, or if paused/resumed, the time we resumed the blueprint.
+	var/time_started = 0
+	/// Controls how fast blueprints are produced. Higher speed settings have a exponential effect on power use.
 	var/speed = DEFAULT_SPEED
 	var/repeat = FALSE
-	var/manual_stop = FALSE
-	var/output_cap = 20
+	var/output_cap = MAX_OUTPUT
+	/// A list of manufacture datums in the form of a queue. Blueprints are taken from index 1 and added at the last index
 	var/list/queue = list()
```

Comment:
Document as code if these are manufacture datums.

`var/list/whatever/the/type/of/the/list/elements/is`